### DISCUSSION TITLE
Planning for 1.0.0a5 features

### DISCUSSION THREAD

**Discussion by shiroinekotfs** [shiroinekotfs/jupyter-cpp-kernel#12] [Announcements]
In the next release, the C++ kernel for Jupyter will have these features I planned so far:

* Install C++ header from PyPI. That sounds weird, but believe me, once the headers can be installed from PyPI and directly interpreted with the C++ kernel, this helps a lot in implementation into the real application
* Minimalize the time of execution of each cell in the C++ kernel
* Remove unnecessary pre-installed headers
* Secure RegEx on adding user I/O
* **Important for Windows user**: Wide support for Windows still continues, but the help by @Welding-Torch for local GCC! Let's welcome him~

Note that not all new features can be implemented in the next release. They may be looked over, because of many various reasons that I will provide once the new version is published.

**Comment by Welding-Torch** [comment]
Thank you for the warm welcome, I am happy to contribute.

**Comment by shiroinekotfs** [comment]
Notification from the developer:

The new C++ kernel for Jupyter, version 1.0.0a5, has been published to PyPI.

Check it out [here](https://pypi.org/project/jupyter-cpp-kernel/1.0.0a5/)

---

For any problems, please visit [Issues](https://github.com/shiroinekotfs/jupyter-cpp-kernel/issues) for the bug report or [Discussions](https://github.com/shiroinekotfs/jupyter-cpp-kernel/discussions) or general talking.

This topic is permanently closed.
